commit/factory.go

```diff
@@ -104,6 +110,32 @@ func (p *PluginFactory) NewReportingPlugin(config ocr3types.ReportingPluginConfi
 		oracleIDToP2PID[commontypes.OracleID(oracleID)] = p2pID
 	}
 
+	// Bind the RMNHome contract
+	var rmnHomeReader reader.RMNHome
+	if rmnEnabled {
+		rmnHomeAddress := p.ocrConfig.Config.RmnHomeAddress
+		rmnCr, ok := p.contractReaders[p.homeChainSelector]
+		if !ok {
+			return nil,
+				ocr3types.ReportingPluginInfo{},
+				fmt.Errorf("failed to find contract reader for home chain %d", p.homeChainSelector)
+		}
+		rmnHomeBoundContract := types.BoundContract{
+			Address: "0x" + hex.EncodeToString(rmnHomeAddress),
```

Comment:
Lets use `typeconv.AddressBytesToString`

Reply:
It will need to be standardized later, better to use it now so it can correctly update all occurrences.